contracts/vaults/BaseVaults/RibbonDeltaVault.sol

```diff
@@ -244,7 +248,7 @@ contract RibbonDeltaVault is RibbonVault, DSMath, OptionsDeltaVaultStorage {
         optionState.nextOptionReadyAt = uint32(block.timestamp.add(delay));
 
         optionState.currentOption = address(0);
-        vaultState.lastLockedAmount = balanceBeforePremium;
+        vaultState.lastLockedAmount = uint104(balanceBeforePremium);
```

Comment:
balanceBeforePremium is already [uint104](https://github.com/ribbon-finance/ribbon-v2/blob/eeeace4dd871e07412fd30aea6fb521414a0ee10/contracts/vaults/BaseVaults/RibbonDeltaVault.sol#L277)

Reply:
Chnaged to uint256

Reply:
then we should change this [line](https://github.com/ribbon-finance/ribbon-v2/blob/eeeace4dd871e07412fd30aea6fb521414a0ee10/contracts/vaults/BaseVaults/RibbonDeltaVault.sol#L277) to balanceBeforePremium = lockedAmount;